sections/semantics-links.include

```diff
@@ -1097,7 +1102,7 @@
       <td><{link/dns-prefetch}></td>
       <td><em>not allowed</em></td>
       <td><a>hyperlink</a></td>
-      <td class="yes"> · </td>
+      <td class="yes"> Yes </td>
       <td>Gives the origin for resources so the User Agent can resolve it as early as possible.</td>
     </tr>
 
```

Comment:
Next row;

```
1109    <tr>
1110      <td<{link/external}></td>
```
→`<td<` is typo.

Reply:
Agree with @mtrootyy . Otherwise lgtm :)